etc/docker/elyra/start-elyra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export NB_PORT=${NB_PORT:-8888}
+export GATEWAY_HOST=${GATEWAY_HOST:-localhost}
+export KG_URL=${KG_URL:-http://${GATEWAY_HOST}:${NB_PORT}}
+export KG_HTTP_USER=${KG_HTTP_USER:-jovyan}
+export KG_REQUEST_TIMEOUT=${KG_REQUEST_TIMEOUT:-120}
+export KERNEL_USERNAME=${KG_HTTP_USER}
+
+echo "Gateway URL: " ${JUPYTER_GATEWAY_URL}
```

Comment:
You're never setting `JUPYTER_GATEWAY_URL`.

Reply:
this should be provided otherwise we will use local kernels